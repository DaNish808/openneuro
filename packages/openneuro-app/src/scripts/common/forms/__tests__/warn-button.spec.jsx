import React from 'react'
import { shallow } from 'enzyme'
import WarnButton from '../warn-button'

describe('common/forms/WarnButton', () => {
  it('renders successfully', () => {
    const wrapper = shallow(<WarnButton message="A Button!" />)
    expect(wrapper).toMatchSnapshot()
  })
  it('renders with warnings disable', () => {
    const wrapper = shallow(<WarnButton message="A Button!" warn={false} />)
    expect(wrapper).toMatchSnapshot()
  })
})
