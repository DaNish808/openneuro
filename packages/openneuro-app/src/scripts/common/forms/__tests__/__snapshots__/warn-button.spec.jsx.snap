// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/forms/WarnButton renders successfully 1`] = `
<span
  className=""
>
  <button
    className="btn-warn-component warning"
    onClick={[Function]}
  >
    <i
      className="fa fa-trash-o"
    />
     
    A Button!
  </button>
</span>
`;

exports[`common/forms/WarnButton renders with warnings disable 1`] = `
<span
  className=""
>
  <button
    className="btn-warn-component warning"
    onClick={[Function]}
  >
    <i
      className="fa fa-trash-o"
    />
     
    A Button!
  </button>
</span>
`;
